muselog/django.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import datetime
```

Comment:
```python 
from datetime import datetime
```
Would be much better since we are only using the datetime object from the module.